help-jellyfin/index.html

```diff
@@ -0,0 +1,439 @@
+<!DOCTYPE html>
```

Comment:
is this generated? if so, where is the source?

Reply:
We don't generate anything yet, @anthonylavado had it on his list I think. All pages are manually created at the moment.

Reply:
I copied a previous page (`downloads/index.html`).